mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageMapper.java

```diff
@@ -144,18 +144,12 @@ public MailboxCounters getMailboxCounters(Mailbox mailbox) throws MailboxExcepti
 
     @Override
     public void delete(Mailbox mailbox, MailboxMessage message) {
-        CassandraId mailboxId = (CassandraId) mailbox.getMailboxId();
-
-        deleteAsFuture(message, mailboxId)
+        deleteAsFuture(message)
```

Comment:
Then the mailbox parameter is no longer in use but yes, it's useful in another API implementation